### PR TITLE
add supported iaas info of IBM Cloud

### LINKED ISSUE #1272: Add clear information about the supported IAAS of IBM Cloud
/kind cleanup
/kind documentation
/area provider/ibmcloud

**Describe the solution you'd like**
- Highlight in book and readme VPC & PowerVS as supported IAAS of IBM Cloud

**Anything else you would like to add:**
NA

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
**What this PR does / why we need it**:
- add supported iaas info of IBM Cloud

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #1272 

**Special notes for your reviewer**:

/area provider/ibmcloud

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note
add supported iaas info of IBM Cloud
```

### CODE REVIEW

**Comment by Karthik-K-N** [comment]
How about inverting the image, Putting CAPIBM at the top and IBMCloud at the bottom. Just thinking we are explaining about capibm and flow should be from top to bottom.

File: docs/book/src/introduction.md [hunk 1]
```diff
@@ -9,14 +9,21 @@
...
 previous cluster managers such as [kops](https://github.com/kubernetes/kops) and
 [kubicorn](http://kubicorn.io/).
 
+## CAPIBM Supported Infrastructure-as-a-Service (IaaS)
+
+<p align="center">
+<img src="../images/ibm-cloud-iaas.png" alt="Supported IBM Cloud IaaS">
+</p>
+
+Currently, the CAPIBM project exclusively facilitates the deployment of Kubernetes (K8s) clusters solely on two IBM infrastructure offerings, namely IBM VPC (Virtual Server Instances) and IBM PowerVS.
```
Comment (mkumatag) [inline docs/book/src/introduction.md:25]: please add the hyperlink to both vpc and powervs

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: docs/book/src/introduction.md (hunk 1 above)
Comment (Prajyot-Parab) [reply docs/book/src/introduction.md]: Done.

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**